docs/examples/good_practices/index.rst

```diff
@@ -1,5 +1,13 @@
-**************
-Good Practices
-**************
+*******************
+Good practices
+*******************
 
-.. include:: /examples/good_practices/checkpointing/_index.rst
+Work-in-progress: This section will contain minimal examples illustrating various good practices
+that should be observed when using the Mila cluster.
```

Comment:
Work-in-progress: This section will contain minimal examples illustrating
various good practices that should be observed when using the Mila cluster.

Reply:
?

Reply:
oups, missing some ticks

Reply:
Fixed in 65d6157897f45be881f5ce33dc1e99eb5bf817fc